src/limbo/opt/cmaes.hpp

```diff
@@ -41,12 +43,12 @@ namespace limbo {
                 int irun, lambda = 0, countevals = 0;
                 char const* stop;
                 boundary_transformation_t boundaries;
-                double lowerBounds[] = {0.0}; // TODO  put this into params?
+                double lowerBounds[] = {0.0}; 
                 double upperBounds[] = {1.006309}; // Allows solution to be pretty close to 1
```

Comment:
Like this? https://en.wikipedia.org/wiki/Fast_inverse_square_root

On 27 Nov 2015, at 12:05, Federico Allocati <notifications@github.com> wrote:

> In src/limbo/opt/cmaes.hpp:
> 
> > @@ -41,12 +43,12 @@ namespace limbo {
> >                  int irun, lambda = 0, countevals = 0;
> >                  char const* stop;
> >                  boundary_transformation_t boundaries;
> > -                double lowerBounds[] = {0.0}; // TODO  put this into params?
> > +                double lowerBounds[] = {0.0}; 
> >                  double upperBounds[] = {1.006309}; // Allows solution to be pretty close to 1
> 
> Binary searched :p
> 
> �
> Reply to this email directly or view it on GitHub.
> 

� Jean-Baptiste Mouret
http://pages.isir.upmc.fr/~mouret/